### ISSUE TITLE
[Deep Learning] Loss Function | 6mini.log

### ISSUE THREAD

**Issue by 6mini** [6mini/6mini.github.io#82]
https://6mini.github.io/deep%20learning/2021/10/19/LossFunc/ 

 손실함수에 대한 의미와 종류 정리(MAE, MSE, RMSE, Binary, Categorical, Sparse Categorical Cross-entropy)
